.github/workflows/test.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Setup .NET Core 5.0 SDK
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '5.0.201'
+          dotnet-version: '5.0.x'
```

Comment:
@Shane32 Do you remember why we install .NET Core 3.1 SDK ?

Reply:
Tests run against both 3.1 and 5.0. I wouldn’t change that as 5.0 is not a LTS version.